AFQ/api.py

```diff
@@ -1734,3 +1726,40 @@ def export_all(self):
             self.viz_ROIs(export_as_gif=True)
             self.export_rois()
         self.combine_profiles()
+
+
+def combine_profiles_in_folder(tract_profile_folder, outfile):
+    """
+    Combine tract profiles from different subjects / sessions
+    into one CSV.
+
+    Parameters
+    ----------
+    tract_profile_folder : path to folder
+        Folder with all tract profiles you wish to merge.
+        Will also search subdirectories recursively.
+        It will select all files of the form '*profiles.csv'.
+    outfile : filename
+        Filename for the output CSV.
+
+    Returns
+    -------
+    Ouput CSV's pandas dataframe.
+    """
+    dfs = []
+    for path in Path(tract_profile_folder).rglob('*profiles.csv'):
```

Comment:
Maybe instead of globing, we could query the BIDS dataset here and infer the well-defined file-names that should be there?

Reply:
I decided to remove this all together, see below.